service/src/test/java/uk/nhs/adaptors/gp2gp/uat/EhrExtractUATTest.java

```diff
@@ -69,7 +69,8 @@
 
 @ExtendWith(MockitoExtension.class)
 public class EhrExtractUATTest {
-    private static final String FILES_PREFIX = "TC4-";
+    private static final String FILES_PREFIX_TC4 = "TC4-";
```

Comment:
Maybe files don't need prefixes at all now, if they're in separate TC4 and TC7 directories?